packages/2d/src/lib/components/Layout.ts

```diff
@@ -996,8 +996,11 @@ export class Layout extends Node {
       const wrap = this.textWrap();
```

Comment:
Can we refactor this whole thing to something like this
```ts
if (this.textWrap.isInitial()) {
  this.element.style.whiteSpace = '';
  return;
}

const wrap = this.textWrap();

if (typeof wrap === 'boolean') {
  this.element.style.whiteSpace = wrap ? 'normal' : 'nowrap';
  return;
}

if (wrap === 'pre') {
  this.element.style.whiteSpace = wrap;
  return;
}

if (wrap === 'balance') {
  this.element.style.whiteSpace = 'normal';
  this.element.style.textWrap = wrap;
  return;
}
```

Reply:
sure, will do